runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargePlugin.java

```diff
@@ -411,6 +423,37 @@ else if (expeditiousCheckMatcher.find())
 			{
 				updateExpeditiousBraceletCharges(Integer.parseInt(expeditiousCheckMatcher.group(1)));
 			}
+			else if (braceletOfClayCheckMatcher.find())
+			{
+				final String match = braceletOfClayCheckMatcher.group(1);
+
+				updateBraceletOfClayCharges(Integer.parseInt(match));
+			}
+			else if (message.equals(BRACELET_OF_CLAY_USE_TEXT))
+			{
+				final ItemContainer equipment = client.getItemContainer(InventoryID.EQUIPMENT);
+
+				// Determine if the player mined clay with a bracelet of clay equipped.
+				if (equipment == null)
+				{
+					return;
+				}
+
+				if (equipment.contains(ItemID.BRACELET_OF_CLAY))
```

Comment:
Instead of two separate checks, combine them.
```suggestion
				// Determine if the player mined clay with a bracelet of clay equipped.
				if (equipment == null || !equipment.contains(ItemID.BRACELET_OF_CLAY))
				{
					return;
				}
```